jupyter/README.md

```diff
@@ -0,0 +1,22 @@
+# Jupyter Kernel
+
+This is a Jupyter Kernel for Swift that is implemented using LLDB's Python APIs.
+
+# Installation Instructions
+
+Create a virtualenv and install jupyter in it.
+```
+virtualenv venv
+. venv/bin/activate
+pip -V # Make sure this says python 2.7. LLDB doesn't support python 3.
+pip install jupyter
```

Comment:
Most machines should have `pip2` command that guarantees python 2, no?

Reply:
Updated !